core/src/commonMain/kotlin/com/ustadmobile/core/controller/Login2Presenter.kt

```diff
@@ -0,0 +1,112 @@
+package com.ustadmobile.core.controller
+
+import com.ustadmobile.core.account.UstadAccountManager
+import com.ustadmobile.core.db.dao.PersonDao
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.impl.AppConfig
+import com.ustadmobile.core.impl.UmAccountManager
+import com.ustadmobile.core.impl.UstadMobileSystemImpl
+import com.ustadmobile.core.view.ContentEntryListTabsView
+import com.ustadmobile.core.view.Login2View
+import com.ustadmobile.core.view.PersonEditView
+import com.ustadmobile.core.view.UstadView.Companion.ARG_NEXT
+import com.ustadmobile.core.view.UstadView.Companion.ARG_SERVER_URL
+import com.ustadmobile.door.DoorDatabaseRepository
+import com.ustadmobile.lib.db.entities.UmAccount
+import io.ktor.client.call.receive
+import io.ktor.client.request.get
+import io.ktor.client.request.parameter
+import io.ktor.client.statement.HttpStatement
+import io.ktor.http.HttpStatusCode
+import io.ktor.http.takeFrom
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Runnable
+import kotlinx.coroutines.launch
+
+class Login2Presenter(context: Any, arguments: Map<String, String>, view: Login2View,
```

Comment:
LoginPresenter should require the UstadAccountManager as a constructor parameter.